tree/dataframe/src/RDataFrame.cxx

```diff
@@ -206,32 +207,37 @@ operations should work with. Here are the most common methods to construct a RDa
 ~~~{.cpp}
 // single file -- all ctors are equivalent
 TFile *f = TFile::Open("file.root");
-auto t = f.Get<TTree>("treeName");
+TTree *t = f.Get<TTree>("treeName");
```

Comment:
Why do you prefer the spelled out type? I like the `auto` more, because it suggest less raw pointer magic and more modern C++.

Reply:
In doubt, I would write `auto tree = f.Get<TTree>("treeName");`

Reply:
`auto` _hides_ raw pointers. in docs, that's not necessarily a good thing. I want to document exactly what type is returned there for clarity.

Reply:
(I also considered using `auto *` but I thought it might throw off users that are not so proficient in C++.